src/osp/Universe.h

```diff
@@ -77,6 +77,11 @@ class Universe
     Universe(Universe &&move) = delete;
     ~Universe() = default;
 
+    /**
+     * Safely destroys the universe
+     */
+    void destroy();
```

Comment:
:)

Reply:
But does it destroy *this* universe? Or the other one?

I choose the universe that was created for Zaphod Beeblebrox

Reply:
This one, hopefully more safely than the Trafalmadorian rocket fuel incident.